TestApp/Package.swift

```diff
@@ -9,7 +9,7 @@ let package = Package(
     .executable(name: "TestApp", targets: ["TestApp"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/kateinoigakukun/JavaScriptKit", .revision("85b8617")),
+    .package(url: "https://github.com/kateinoigakukun/JavaScriptKit", .revision("c90e82f")),
   ],
   targets: [
     // Targets are the basic building blocks of a package. A target can define a module or a test suite.
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: TestApp/Package.swift;
  Line: 15;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 104 characters (`line_length`)